test/werker.test.js

```diff
@@ -97,7 +102,7 @@ it.describe("werker",function (it) {
                     setTimeout(function () {
                         assert.lengthOf(closes, 4);
                         next(null);
-                    }, 1000);
+                    }, 2000);
```

Comment:
@doug-martin this is what I was saying I had to change in werker. This was failing on Travis.